packages/client/components/ErrorComponent/ErrorComponent.tsx

```diff
@@ -27,6 +27,21 @@ const ErrorComponent = (props: Props) => {
   const {error, eventId} = props
   console.error(error)
   const {modalPortal, openPortal, closePortal} = useModal()
+  const oldBrowserErrs = ['flatMap is not a function']
+
+  const handleClick = () => {
+    const url = 'https://browser-update.org/update-browser.html'
+    window.open(url)
```

Comment:
must use `_blank, noreferrer` for security reasons.

consider using `<a>` instead of onClick to be a little more declarative